src/locales/de/translation.json

```
@@ -54,7 +54,7 @@
   },
   "reset-pw": {
     "title": "Passwort zurücksetzen",
-    "subtitle": "Gib dein altes Passwort ein und wähle ein neues.",
+    "subtitle": "Gib dein altes Passwort ein und wähle ein Neues.",
```

While add it could you fix this?
```suggestion
    "subtitle": "Wähle ein neues Passwort.",
```